src/main/java/Exceptions/InvalidInputException.java

```diff
@@ -0,0 +1,7 @@
+package Exceptions;
+
+public class InvalidInputException extends LeluException{
+    public InvalidInputException(String message) {
+        super(message);
+    }
+}
```

Comment:
I like how you have created even more specific exceptions ontop of LeluException